### PR TITLE
Add code field

### PR DESCRIPTION
**Pull request by PaulBunker**
# Code field
This is the scaffolding for implementing a 'code editor' field for tina

## why?
Some people would rather use raw markdown than the wysiwym field. This is currently just a textfield with monospace font but could be extended.

### Issues
This field works a deam in the about page in the sidebar but is not populated with content when used 'inline' not sure why. Can you help?

#### Tina commits
I'm sorry about the tina junk commits. Can we have a switch to stop her doing that as well as pushing?

### CODE REVIEW

File: packages/@tinacms/fields/stories/code.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+*/
+
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Code } from '../src/Code'
+
+storiesOf('Code', module)
+  .add('Default', () => <Code />)
+  .add('Auto height', () => <Code />)
```
Comment (ncphillips) [inline packages/@tinacms/fields/stories/code.stories.tsx]:
```suggestion
```

**Comment by ncphillips** [comment]
@PaulBunker did you say there was an issue with this? I can take a look at it this week and get merged soon.

**Comment by PaulBunker** [comment]
I couldn't get it to work 'inline' no clue why. I had intended to take another look but I had some urgent issues to deal with so this took a back seat. Then I went camping 🏕

**Comment by maciekgrzybek** [comment]
I just spoke with @PaulBunker and I can try to help out with this ticket.

**Comment by PaulBunker** [comment]
Closing for now, I have a local plugin. Not sure if this is useful for anyone but myself.

**Pull request closed without merging**